src/components/WPResponsiveImage/WPResponsiveImage.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Image } from '..';
+import { arrayFromRange, resizeWPImage } from '@quartz/js-utils';
+
+function WPResponsiveImage( {
```

Comment:
i'd probably just call it `ResponsiveImage`. while i get it as a cue to the engineer to make sure the image is hosted in our media library, other prism components will consume this component and render that cue invisible. (see e.g. `Badge`). also fwiw, wordpress isn't doing the resizing, it's a special service that VIP provides.